content/ADR-138-social-service-authentication.md

```diff
@@ -0,0 +1,101 @@
+---
+adr: 138
+date: 2022-11-28
+title: Social service authentication
+authors:
+  - jmoguilevsky
+status: Draft
+type: RFC
+spdx-license: CC0-1.0
+---
+
+## Abstract
+
+<!--
+Insert here a brief paragraph summarizing the RFC in its current state.
+This section shall provide an overview of whether this is a settled
+decision, alternatives explored and a short summary of relevant
+background information and key insights.
+-->
+
+As part of the social service initiative users will need to be able to authenticate themselves to be able to interact with other users and be able to access their information correctly.
```

Comment:
The abstract does not describe the current status of the document nor summarizes the content of it.

Reply:
Updated the abstract